spec.html

```diff
@@ -36501,22 +36481,12 @@ <h1>%AsyncFromSyncIteratorPrototype%.throw ( _value_ )</h1>
             1. If _throw_ is *undefined*, then
               1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; _value_ &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Let _throwResult_ be Call(_throw_, _syncIterator_, &laquo; _value_ &raquo;).
-            1. IfAbruptRejectPromise(_throwResult_, _promiseCapability_).
-            1. If Type(_throwResult_) is not Object, then
+            1. Let _result_ be Call(_throw_, _syncIterator_, &laquo; _value_ &raquo;).
+            1. IfAbruptRejectPromise(_result_, _promiseCapability_).
+            1. If Type(_result_) is not Object, then
               1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Let _throwDone_ be IteratorComplete(_throwResult_).
-            1. IfAbruptRejectPromise(_throwDone_, _promiseCapability_).
-            1. Let _throwValue_ be IteratorValue(_throwResult_).
-            1. IfAbruptRejectPromise(_throwValue_, _promiseCapability_).
-            1. Let _valueWrapperCapability_ be ! NewPromiseCapability(%Promise%).
-            1. Perform ! Call(_valueWrapperCapability_.[[Resolve]], *undefined*, &laquo; _throwValue_ &raquo;).
-            1. Let _steps_ be the algorithm steps defined in <emu-xref href="#sec-async-from-sync-iterator-value-unwrap-functions" title></emu-xref>.
-            1. Let _onFulfilled_ be CreateBuiltinFunction(_steps_, &laquo; [[Done]] &raquo;).
-            1. Set _onFulfilled_.[[Done]] to _throwDone_.
-            1. Perform ! PerformPromiseThen(_valueWrapperCapability_.[[Promise]], _onFulfilled_, *undefined*, _promiseCapability_).
-            1. Return _promiseCapability_.[[Promise]].
+            1. Return AsyncFromSyncIteratorContinuation(_result_, _promiseCapability_).
```

Comment:
Should all these calls be prefixed with a `?` or a `!`?

Reply:
Thanks for the comment!

After some help from @mathiasbynens sounded like prefixing with "!" is the right thing to do (since the called function contains "!" calls as well).

Reply:
Seems right to me!